### DISCUSSION TITLE
How to broadcast an intent during a flow?

### DISCUSSION THREAD

**Discussion by andre-artus** [mobile-dev-inc/Maestro#174] [Q&A] [answered]
Many of my Android apps have a use-case that is initiated via a broadcast (i.e. implements a `BroadcastReceiver`).

When testing manually I can send the broadcast via `adb`, is there a way to either call an `adb` command or otherwise initiate a broadcast using Maestro?

**Comment by ArthurSav** [comment, marked as answer]
Hi @andre-artus, thanks for the suggestion. That is not supported at the moment but it would be a great addition. Would you mind opening a [feature request](https://github.com/mobile-dev-inc/maestro/issues/new?assignees=&labels=enhancement&projects=&template=feature_request.md&title=%5BFeature+Request%5D) and we'll tackle it when possible.

**Reply by andre-artus** [reply]
I'll do that.
